docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,197 @@
+
+A **BigchainDB Network** (*Network* henceforth) is a set of **4 or more BigchainDB Nodes** (*Node* henceforth). Every Node is independently managed by a Member, and runs an instance of the [BigchainDB Server software][bdb:software]. At the **Genesis** of a Network, there **MUST** be at least **4** Nodes ready to connect. After the Genesis, a Network can dynamically add new Nodes or remove old Nodes.
+
+A Network will stop working if more than one third of the Nodes are down. The bigger a Network, the more failures it can handle. A Network of size 4 can tolerate only 1 failure, so if 3 out of 4 Nodes are online, everything will work as expected. Eventually, the Node that was offline will automatically sync with the others.
+
+
+## Before we start
+This tutorial assumes you have basic knowledge on how to manage a GNU/Linux machine. The commands are tailored for an up-to-date *Debian-like* distribution (we use an **Ubuntu 17.10** Virtual Machine on Microsoft Azure): if you are on a different distribution you might need to adapt the name of the packages installed.
+
+We don't make any assumptions on **where** you run the Node.
+You can run the BigchainDB Server on a Virtual Machine on the cloud, on a machine in your data center, on a Raspberry Pi. Just make sure that your Node is reachable by the other Nodes, a **non-exhaustive list of examples**:
+- **good**: all Nodes running in the cloud using public IPs.
+- **bad**: some Nodes running in the cloud using public IPs, some Nodes in a private network.
+- **good**: all Nodes running in a private network.
+
+The rule of thumb is: if Nodes can ping each other, then you are good to go.
+
+The next sections are labelled with **Member** or **Coordinator**, depending on who should follow the instructions. Remember, a Coordinator is also a Member.
+
+## Member: Set up a Node
+Every Member in the Network **must** set up its own Node. The process consists of installing three components, BigchainDB Server, Tendermint Core, and MongoDB, and configure the firewall.
+
+**Important note on security: it's up to the Member to harden their system.**
+
+### Install the required software
+
+#### Install BigchainDB Server
+BigchainDB Server requires **Python 3.6+**, make sure your system has it. Install the required packages:
+
+```bash
+sudo apt install -y python3-pip libssl-dev
+```
+
+Now install the latest version of BigchainDB. Check the [project page on PyPI][bdb:pypi] for the last version (`2.0.0a5` at the time of writing) and install it:
+
+```bash
+sudo pip3 install bigchaindb==2.0.0a5
+```
+
+Check you installed the correct version of BigchainDB Server with `bigchaindb --version`.
+
+#### Install MongoDB
+Install a recent version of MongoDB. BigchainDB Server requires `3.4` or newer.
+
+```bash
+sudo apt install mongodb
```

Comment:
After doing this, I checked the MongoDB version:
```text
$ mongod --version
db version v3.6.3
git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
OpenSSL version: OpenSSL 1.1.0g  2 Nov 2017
allocator: tcmalloc
modules: none
build environment:
    distarch: x86_64
    target_arch: x86_64
```
Uh oh, it's MongoDB 3.6.3, but we only test BigchainDB on MongoDB 3.4.x. I'm not sure what to do here: We could start testing on MongoDB 3.6.x, or we could force the install of MongoDB 3.4.x.

Reply:
I started looking up how to install MongoDB 3.4.x on Ubuntu and found the page https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/ 

It says:
![screenshot from 2018-05-17 11-41-06](https://user-images.githubusercontent.com/1210951/40169898-5e2628b6-59c7-11e8-8596-0eca02ca2221.png)

In other words, we shouldn't be using `sudo apt install mongodb`

Reply:
It seems MongoDB (the company) doesn't have an official `mongodb-org` 3.4.x package for Ubuntu 18.04 LTS (bionic) yet, as far as I can tell. I tried it but ran into this error message:

"The repository 'http://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/3.4 Release' does not have a Release file."

Maybe the solution, for now, is to use Ubuntu 16.04 LTS.

Reply:
Good point, I see in our `setup.py` that we are using an old version of `pymongo`. I'll make a PR to upgrade to the last version of the driver, that is compatible with [all versions of MongoDB](https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#reference-compatibility-mongodb-python).

Reply:
Okay, but what version of MongoDB should we be installing here: 3.4 or 3.6?

I suppose we _could_ use Ubuntu 18.04 and `sudo apt install mongodb` (i.e. the MongoDB package from Canonical, not from the MongoDB company) because that installs MongoDB 3.6.3 (currently).

I'd _prefer_ to use the official `mongodb-org` package from MongoDB, but they don't have a version for Ubuntu 18.04 yet.

I think the best option, for now, is to use Ubuntu 16.04 LTS and to use the official `mongodb-org` package to install MongoDB 3.6 on that.